app/css/app.css

```diff
@@ -1,52 +1,20 @@
+/** General styles **/
 html {
+  height: 100%;
+
   font: normal 10px/10px Arial;
 }
 
 body {
+  height: 100%;
   margin: 0;
   padding: 0;
+
+  background: #FFF;
```

Comment:
Please lower case for colours, or better in this case, use `white` (easier to read and understand for us poor human being)